### PR TITLE
feat: style added for button component

### PR DESCRIPTION
**Pull request by raman1999**
image

### CODE REVIEW

**Review by krishanu2911 — COMMENTED** [review]
everything looks good to go No changes required according to me 
it's good that you have used DRY and CSS-variable

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+.btn-light:hover{
+    background-color: rgba(var(--bgLight),0.5);
+}
+
+.btn-dark {
+    color: var(--txtDark);
+    background-color: var(--bgDark);
+    border: 2px solid var(--txtDark);
```
Comment (Sameony) [inline components/button.css:50]: you can add the border property to the btn class and specify the colors in the respective sub-classes

**Review by Sameony — COMMENTED** [review]
Suggesting Minor Adjustments

File: components/button.css (hunk 1 above)
Comment (raman1999) [reply components/button.css:50]: Thanks for previewing. Actually I have  used this classes in other component also , that's why I separately write border in each sub-class

**Pull request merged**